runtime/doc/autocmd.txt

```diff
@@ -994,7 +994,7 @@ expect.
 Environment variables can be used in a pattern: >
 	:autocmd BufRead $VIMRUNTIME/doc/*.txt  set expandtab
 And ~ can be used for the home directory (if $HOME is defined): >
-	:autocmd BufWritePost ~/.vimrc   so ~/.vimrc
+	:autocmd BufWritePost ~/.config/nvim/init.vim   so <afile>
```

Comment:
17.10.2015, 22:01, "Michael Reed" <notifications@github.com>:
> In runtime/doc/autocmd.txt:
>
>> @@ -994,7 +994,7 @@ expect. > Environment variables can be used in a pattern: > > :autocmd BufRead $VIMRUNTIME/doc/*.txt set expandtab > And ~ can be used for the home directory (if $HOME is defined): > > - :autocmd BufWritePost ~/.vimrc so ~/.vimrc > + :autocmd BufWritePost ~/.config/nvim/init.vim so <afile>
>
> Regarding all these references to ~/.config/...: shouldn't these instead say $XDG_CONFIG_HOME/...?

Not all of them: $XDG_CONFIG_HOME is not guaranteed to be defined since PR with approach which adds support for XDG_* vars to vim_getenv was rejected.

Most of other stay as-is because I am not fond of explaining WTF is $XDG_CONFIG_HOME and what is its default. I guess we need another section somewhere in documentation (options.txt?) with explanations.

>
> —
> Reply to this email directly or view it on GitHub.